From fcc3e71595d0dfd32f0c21aec4ce0a6b5667ccb5 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Wed, 17 Mar 2021 17:41:02 +0100
Subject: [PATCH 26/35] Disable the 'cygwin' GitHub workflow

It does not work at all. For example, `rpm -E %fedora` says that there
should be version 33 of rpmsphere at
https://github.com/rpmsphere/noarch/tree/master/r, but there is only
version 32.

Another thing that is broken: Cygwin now assumes that a recent
mingw-w64-headers version is available, but Fedora apparently only
offers v7.0.0, which is definitely too old to accommodate for the
expectation of https://github.com/cygwin/cygwin/commit/c1f7c4d1b6d7.

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 .github/workflows/cygwin.yml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.github/workflows/cygwin.yml b/.github/workflows/cygwin.yml
index f9a9a7ae5b..4b541a887a 100644
--- a/.github/workflows/cygwin.yml
+++ b/.github/workflows/cygwin.yml
@@ -1,6 +1,6 @@
 name: cygwin
 
-on: push
+on: workflow_dispatch
 
 jobs:
   fedora-build:
-- 
2.36.0

